org.gnome.Cheese.appdata.xml.in.patch.1

```diff
@@ -1,16 +1,77 @@
 --- a/data/org.gnome.Cheese.appdata.xml.in
 +++ b/data/org.gnome.Cheese.appdata.xml.in
-@@ -37,4 +37,13 @@
+@@ -37,4 +37,74 @@
    <update_contact>davidk@gnome.org</update_contact>
    <project_group>GNOME</project_group>
    <translation type="gettext">cheese</translation>
 +  <content_rating type="oars-1.1"/>
 +  <releases>
 +    <release version="3.34.0" date="2019-09-10">
-+      <url>https://gitlab.gnome.org/GNOME/cheese/raw/3.34.0/NEWS</url>
-+    </release>
-+    <release version="3.32.0" date="2019-03-12">
-+      <url>https://gitlab.gnome.org/GNOME/cheese/raw/3.32.0/NEWS</url>
++      <description>
++        <p>
++          <ul>
++            <li>
++              build: Don't use 'lib' prefix in find_library
++              Meson passes it directly to -l option, so an extra 'lib' can cause error
++              because it tells the linker to find liblibcanberra-gtk3.so.
++            </li>
++            <li>
++              Add translator comments to the desktop file
++            </li>
++            <li>
```

Comment:
The changelogs are for regular users to read, this massive list is not helpful.

Reply:
Not sure how to handle this. Should a single bullet be used to indicate that this version has seen translation improvements? It was suggested I implement a <description> tag for the release. The best source of information specific to 3.34 for Cheese was the NEWS file. This is little more than a copy paste of it's content.

Advice?

Reply:
Honestly probably just remove the updated translations bullet point, its not interesting.

Reply:
Dropped the last major list item about translations, should I drop the other as well? 415264e93ce95b9efb4624b8e8a2fe23e6962531

Thanks.